notebooks/Using Orion on multivariate input.ipynb

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
Did not fully check this file. But I know there is a way to automatically test code snippets in notebooks (refer to sdv library). Not sure whether it is worthwhile to have a similar one here.

Reply:
Good suggestion, I will add it in another PR to keep them separate. 
Follow #251.